### PR TITLE
Ignore test screenshots folder in .gitignore

### PR DESCRIPTION
**Pull request by CaCTuCaTu4ECKuu**
* Ignore test screenshots folder in .gitignore

* Refactor EvaluationTests.ShouldAcceptObjectHandleAsAnArgument

Change variable type to object as test suggest

### CODE REVIEW

**Review by kblok — COMMENTED** [review]
Shouldn't this be the same?

**Comment by CaCTuCaTu4ECKuu** [comment]
Yes, I dont know why have such a test in a first place but who am I to decide :)

File: lib/PuppeteerSharp.Tests/EvaluationTests/PageEvaluateTests.cs [hunk 1]
```diff
@@ -323,7 +323,7 @@ public async Task ShouldProperlySerializeNullFields()
         public async Task ShouldAcceptObjectHandleAsAnArgument()
         {
             await Page.SetContentAsync("<section>42</section>");
-            var element = await Page.QuerySelectorAsync("section");
+            object element = await Page.QuerySelectorAsync("section");
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/EvaluationTests/PageEvaluateTests.cs:326]: We don't need this. I can merge this PR if it's only about the gitignore.

Comment (CaCTuCaTu4ECKuu) [reply lib/PuppeteerSharp.Tests/EvaluationTests/PageEvaluateTests.cs]: Ok
Anyway what was that test even about? It sounds straightforward but I dont quiet get what it's testing. Should it be removed?

**Pull request merged**